command.go

```diff
@@ -725,12 +725,14 @@ func (cmd *ScanCmd) readReply(cn *conn) error {
 
 //------------------------------------------------------------------------------
 
+// TODO: rename to ClusterSlot
 type ClusterSlotInfo struct {
 	Start int
 	End   int
 	Addrs []string
```

Comment:
@dim new cluster-slots returns following reply

```
127.0.0.1:30001> cluster slots
1) 1) (integer) 0
   2) (integer) 5460
   3) 1) "127.0.0.1"
      2) (integer) 30001
      3) "09dbe9720cda62f7865eabc5fd8857c5d2678366"
   4) 1) "127.0.0.1"
      2) (integer) 30004
      3) "821d8ca00d7ccf931ed3ffc7e3db0599d2271abf"
```

I.e. it contains node id which requires us to change struct definition to something like

```
type ClusterNode struct {
    Addr string
    Id string
}

type ClusterSlot struct {
 	Start int
 	End   int
 	Nodes []ClusterNode
}
```

What do you think? Should we do it in redis.v4 or it is fine do it in redis.v3?